modules/virtualization/microvm/guivm.nix

```diff
@@ -57,6 +57,7 @@
             pkgs.waypipe
             pkgs.networkmanagerapplet
             pkgs.nm-launcher
+            pkgs.mitmweb-ui
```

Comment:
Maybe this would be better behind a debug option.

Reply:
Resolved by [0e2656b](https://github.com/tiiuae/ghaf/commit/0e2656bf1a00ef6752bcbf2781aa5ae7f204b685)